pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -127,16 +127,54 @@ message NamespaceOption {
     optional bool host_ipc = 3;
 }
 
+// PodSecurityContext holds pod-level security configuration that will be applied to a container.
```

Comment:
The comments indicates both can happen (e.g. for selinux options, container level overrides the pod level, for run_as_user, the pod one overrides the container one)